docs/src/development/local/docksal.md

```diff
@@ -1,128 +1,200 @@
+{{< /codetabs >}}
+
+## 6. Run your project
+
+Now your project is ready to run.
+In your project directory, run `fin init`.
+
+The command returns the project URL `http://<DIRECTORY_NAME>.docksal/`.
+Open this URL to see your project running.
+
+## What's next
+
+You've got a project running on a local web server.
+Now you can add customizations.
+For more ideas and options, see the [Docksal documentation](https://docs.docksal.io/).
+
+### Add environment variables
+
+If your project requires environment variables for its hooks,
+add them to the Docksal environment.
+
+{{< codetabs >}}
+
+---
+title=Using the CLI
+file=none
+highlight=bash
+---
+
+fin config set <VARIBLE_NAME>=<VARIABLE_VALUE>
 
-By default, Docksal comes configured with a PHP 7.1 container, an Apache 2.4 web container, and a MySQL 5.6 database container.
-Additional versions are available in the images
-and you can set the desired versions by setting following variables within your `.docksal/docksal.env` file.
+<--->
 
-```text
-# Apache Versions 2.2 / 2.4
-#WEB_IMAGE='docksal/web:2.1-apache-2.2'
-WEB_IMAGE='docksal/web:2.1-apache-2.4'
+---
+title=Directly in the configuration file
+file=none
+highlight=false
+---
 
-# MySQL Version: 5.6 / 5.7 / 8.0
-#DB_IMAGE='docksal/db:1.2-mysql-5.6'
-DB_IMAGE='docksal/db:1.2-mysql-5.7'
-#DB_IMAGE='docksal/db:1.2-mysql-8.0'
+```txt {location=".docksal/docksal.env"}
+<VARIBLE_NAME>=<VARIABLE_VALUE>
+
+HOSTING_PLATFORM="platformsh"
+```
 
-# PHP Versions Available 5.6 / 7.0 / 7.1 / 7.2
-#CLI_IMAGE='docksal/cli:2.5-php5.6'
-#CLI_IMAGE='docksal/cli:2.5-php7.0'
-CLI_IMAGE='docksal/cli:2.5-php7.1'
-#CLI_IMAGE='docksal/cli:2.5-php7.2'
+{{< /codetabs >}}
+
+To apply your changes, run the following command:
+
+```bash
+fin project restart
 ```
 
-You can further create and customize a `.docksal/docksal.yml` file within your project.
-This is a docker-compose file and can be customized as needed for your application,
-as some customizations are specific to certain applications.
-See Docksal documentation on [extending stock images](https://docs.docksal.io/stack/extend-images).
+### Customize your stack
 
-## Downloading MySQL data from Platform.sh into Docksal
+Docksal comes configured with a default PHP container (known as the CLI container),
+web container, and database container.
+You can customize the versions for each.
 
-In most cases, downloading data from Platform.sh and loading it into your project is straightforward.
-The following commands, run from your application root, download a compressed database backup
-and load it into the local Docksal database container.
+Check the available versions for each by running `fin image registry <IMAGE>`.
+For example, to check the available containers for the app, run `fin image registry docksal/cli`.
+
+You can then use the version you want by adding it as an [environment variable](#add-environment-variables).
+For example, to use the latest version, run the following command:
```

Comment:
💅
A bullet list of examples might be nicer to read?

Reply:
I'm not sure I understand. There's one example, so it doesn't seem to fit well into a bullet list. There are two steps, but one example for each.